ppr-ui/src/components/common/ContactInformation.vue

```diff
@@ -0,0 +1,344 @@
+<template>
+  <div id="contact-info-container">
+    <h2>{{ content.title }}</h2>
+    <p class="mt-2">{{ content.description }}</p>
+    <v-card
+      id="contact-info"
+      flat
+      rounded
+      class="mt-8 pa-8 pr-6"
+      :class="{ 'border-error-left': validateContactInfo }"
+    >
+      <v-row no-gutters>
+        <v-col cols="12" sm="2">
+          <label
+            class="generic-label"
+            :class="{'error-text': validateContactInfo}"
```

Comment:
Super minor but spacing should be consistent within the {}. 
I'm good with either 👍